csrc/preseg_passes/move_pad.cpp

```diff
@@ -533,8 +533,14 @@ void replaceCat(Fusion* fusion) {
   std::vector<Expr*> exprs = fusion->exprs();
 
   // sanitizing CatOp with series of binary add
-  for (auto* cat : ir_utils::filterByType<CatOp>(exprs)) {
+  // NOTE: `.vector()` shouldn't be needed here. But we hit a segfault issue on
+  // a test build environment when iterating FilteredView. This allows us to WAR
+  // the segfault for now. Tracking the issue internally
+  for (auto* cat : ir_utils::filterByType<CatOp>(exprs).vector()) {
```

Comment:
I'm not sure why FilteredView is segfaulting on that strange build. But at least converting it to a vector hide the issue.

cc'ing @xwang233 in case you know something about that build on top of your head.

Reply:
nvm, this is a real issue as @jacobhinkle pointed out that this is a generator